guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+%}
+
+# Silent payments
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+Bitcoin addresses are only meant to be used once. Thus, transaction flows usually start with users interacting with each other to specify the on-chain address(es) to be used in the transaction, with the goal of preventing on-chain address reuse. This not only adds steps to the user flow and adds friction by way of time and mental overhead on the part of users. BIP-352 specifies a protocol called Silent Payments that allows users to avoid such an interaction altogether, while ensuring on-chain addresses are not reused through a new type of static payment addresses.
+
+For example, Alice, who runs an NGO, can seek bitcoin donations from her supporters by posting a static address on her website, and receive donations at unique on-chain addresses without any other interaction with the donors.
+
+This is a significant improvement in user experience for on-chain transactions. Additionally, BIP-352 allows users to proactively add labels to their addresses that get recognised while payments are detected. This is a boost for coin control as well as the contacts features and can improve privacy.
+
+We will first look at Labels and Contacts, then delve into how they impact/improve other features.
+
+
+## Brief explanation
+
+A silent payment transaction happens in 4 broad steps:
+- The receiver shares/publishes a static payment address
+- The sender obtains & uses it to derive a unique on-chain address
+- The sender broadcasts a transaction that pays this derived address
+- The receiver identifies the payment as theirs by identifying that they control this address
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   alt-text = "An image that illustrates how silent payments work in 4 steps."
+   caption = "."
+   width = 1024
+   height = 522
+   layout = "full-width"
+%}
+
+With this new model of transacting, BIP-352 introduces new primitives and concepts for users and developers. Below is a brief summary of them:
+
+| Current terminology        | With silent payments          | Benefit |
+|:-------------|:------------------|:------|
+| Single-use on-chain address           | Static address | Reusable  |
+| Extended private key (xprv) | Spend key   |   |
+| Extended public key (xpub)           | Scan key      | Addresses cannot be deterministically derived   |
+| **Address-based model**           | **Contact-based model** | **More intuitive and familiar**  |
+
+{% include image-gallery.html pages = page.images_create %}
+
+
+## Contacts & labels
```

Comment:
Can contacts and labels maybe just be their own sections instead of having a combo section with two sub-sections?

Reply:
I'm very open to this, what would be the improvement by doing this?